File: src/main/java/org/nette/neon/lexer/NeonTokenType.java
package org.nette.neon.lexer;

import com.intellij.psi.tree.IElementType;
import org.nette.neon.NeonLanguage;
import org.jetbrains.annotations.NotNull;


public class NeonTokenType extends IElementType {
	public NeonTokenType(@NotNull String debugName) {
		super(debugName, NeonLanguage.INSTANCE);
	}

	public String toString() {
		return "[Neon] " + super.toString();
	}
}

File: src/main/kotlin/org/nette/neon/lexer/NeonTokenType.kt
package org.nette.neon.lexer

import com.intellij.psi.tree.IElementType
import org.nette.neon.NeonLanguage

class NeonTokenType(debugName: String) : IElementType(debugName, NeonLanguage.INSTANCE) {
    override fun toString(): String {
        return "[Neon] " + super.toString()
    }
}
